libqpdf/QPDFObjectHandle.cc

```diff
@@ -2080,6 +2081,7 @@ QPDFObjectHandle::parseInternal(PointerHolder<InputSource> input,
                 object = QPDFObjectHandle(new QPDF_Array(olist));
                 setObjectDescriptionFromInput(
                     object, context, object_description, input, offset);
+                object.setParsedOffset(offset - 1);
```

Comment:
I'm not sure this handles whitespace properly.

Reply:
If I understand correctly, whitespace is not handled here.
`offset` points to the next character of the beginning of the array "[".
By `offset - 1`, the parsed offset is changed to point to the "[".
If you prefer the parsed offset to point the next character of the "[", I'll fix so.